packages/perseus/src/widgets/interactive-graphs/interactive-graph-question-builder.ts

```diff
@@ -288,21 +402,33 @@ interface InteractiveFigureConfig {
 
 class SegmentGraphConfig implements InteractiveFigureConfig {
     private numSegments: number;
+    private coords: CollinearTuple[];
     private startCoords?: CollinearTuple[];
 
-    constructor(numSegments: number, startCoords?: CollinearTuple[]) {
-        this.numSegments = numSegments;
-        this.startCoords = startCoords;
+    constructor(options?: {
+        numSegments?: number;
+        startCoords?: CollinearTuple[];
+        coords?: CollinearTuple[];
+    }) {
+        this.numSegments =
```

Comment:
Not a ship blocker: Should this throw an error if these are not the same length?

Reply:
I keep going back and forth on this. I almost want to say no, because if we were to add it here, we would also have to add errors for any difference in length between coords and startCoords for all the other graph types that have a coord array.

I kind of want to just trust the devs, but maybe @benchristel has stronger opinions on this.

Reply:
I'm inclined not to add any validations until we mess up a test and we're like "gosh I wish the builder had validations". The risk incurred by doing nothing seems low.